Sources/PrimerSDK/Classes/Core/Analytics/Analytics.swift

```diff
@@ -32,10 +32,11 @@ class Analytics {
         var primerAccountId: String?
         var properties: AnalyticsEventProperties? = nil
         var sdkSessionId: String
-        var sdkType: String = PrimerSource.sdkSourceType.sourceType
+        var sdkType: String
         var sdkVersion = Bundle.primerFramework.releaseVersionNumber
         var sdkIntegrationType: PrimerSDKIntegrationType? = PrimerInternal.shared.sdkIntegrationType
         var sdkPaymentHandling: PrimerPaymentHandling? = PrimerSettings.current.paymentHandling
+        var integrationType: String
```

Comment:
We should try to avoid modifying the schema of the main analytics object 🤔 
Are you sure this is surfaced on metabase?

Reply:
It's in the body of the event in Metabase, it's not mapped though at the moment (along with other vars). The above has been discussed and agreed, and it's already there in the released RC.

Reply:
thanks!